benches/bloom.rs

```diff
@@ -0,0 +1,101 @@
+#![feature(test)]
+
+extern crate test;
+use bv::BitVec;
+use fnv::FnvHasher;
+use solana::bloom::{Bloom, BloomHashIndex};
+use solana_sdk::hash::{hash, Hash};
+use solana_sdk::signature::Signature;
+//use std::collections::HashSet;
+use hashbrown::HashSet;
+use std::hash::Hasher;
+use test::Bencher;
+
+#[bench]
+#[ignore]
```

Comment:
why are these bench ignore?

Reply:
'cuz our infra doesn't know about 'em and crashes when presented with results for comparison?  I dunno...  maybe @mvines can answer...  this bench is only interesting in 2 dimensions and only interesting when compared to a separate baseline (hashbrown), so is ok to stay human?